README.md

```diff
@@ -18,31 +18,14 @@
 
 ![Hero Image with Logo](./.github/hero2.png)
 
-[React Native](https://reactnative.dev) enables you to build world-class application experiences on native platforms using a consistent developer experience based on JavaScript and [React](https://reactjs.org/). The focus of React Native is on developer efficiency across all the platforms you care about - learn once, write anywhere. Facebook uses React Native in multiple production apps and will continue investing in React Native.
+> See the official [React Native website](https://reactnative.dev/) for an introduction to React Native.
+
+[React Native](https://reactnative.dev) is a framework developed by Facebook that enables you to build world-class application experiences on native platforms using a consistent developer experience based on JavaScript and [React](https://reactjs.org/). The focus of React Native is on developer efficiency across all the platforms you care about - learn once, write anywhere.
 
 This repository adds support for the [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads), which allows you to build apps for [all devices supported by Windows 10](https://developer.microsoft.com/en-us/windows/get-started-windows-10) including PCs, tablets, 2-in-1s, Xbox, Mixed reality devices etc.
 
 ## Status and roadmap
-
-We are in the process of re-implementing react-native-windows in C++, for better performance, and to better align with the shared C++ react-native core as it evolves. Please see the [Roadmap 2019](https://microsoft.github.io/react-native-windows/blog/2019/07/22/roadmap) blog for details on this effort and investment roadmap for the upcoming months.
-
-Please see the [July 2019 Updates Blog](https://microsoft.github.io/react-native-windows/blog/2019/07/21/updates) for all the latest updates!
-
-## Quick Links
-
-- [Getting Started](#getting-started)
-- [Getting Help](#getting-help)
-- [Documentation](#documentation)
-- [Examples](#examples)
-- [Extending React Native](#extending-react-native)
-- [Opening Issues](#opening-issues)
-- [Contributing](#contributing)
-- [License](#license)
-- [Code of Conduct](#code-of-conduct)
-
-## Introduction
-
-See the official [React Native website](https://reactnative.dev/) for an introduction to React Native.
+[Check out our blog](https://microsoft.github.io/react-native-windows/blog/) If you'd like to stay up to date on the status of React Native for Windows and check out current and past roadmaps,  We will post all new releases, updates and general news about the project there.
 
 ## System Requirements
```

Comment:
Do we need this section here anymore? Or can we just link to the docs? https://microsoft.github.io/react-native-windows/docs/rnw-dependencies

Otherwise, starting with 62, we will require 2019 to work on RNW. It's still not clear if 062 RNW apps will require 2019, I think it'll depend on if/when we get the Microsoft.ReactNative nuget package published.

Reply:
I think it makes more sense to link to the docs, and there we can specify actual versions.

Reply:
I'm still seeing a *System Requirements* section starting at line 30, which says to download VS 2017or greater. I'm saying, don't say any versions in this readme, just link out to the docs page here: https://microsoft.github.io/react-native-windows/docs/rnw-dependencies

That way we don't have requirements in two places that we need to keep in sync.

Reply:
Ah! Gotcha.